inscriptions/ISic000483.xml

```diff
@@ -186,23 +192,36 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/><name>Furiae</name> <g ref="#interpunct">·</g> <name>Sa
+                    <lb n="2" break="no"/>biniae</name> <g ref="#interpunct">·</g> <name>Tran
+                    <lb n="3" break="no"/>quillinae</name>
+                    <lb n="4"/><expan><abbr>aug</abbr><ex>ustae</ex></expan> <g ref="#interpunct">·</g> coniugi 
```

Comment:
capitalise Augustae? (wife of emperor, part of her name / titulature)
https://en.wikipedia.org/wiki/Tranquillina